src/emulator/gui/src/game_selector.cpp

```diff
@@ -179,20 +181,61 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
+                        gui.title_id = game.title_id;
                 }
+
+                if (ImGui::BeginPopupContextItem("#game_context_menu")) {
+                    if (gui.title_id == game.title_id) {
```

Comment:
Unhealthy. Isnt there a machinism to detect what right click is pressed on what game without checking title id

Reply:
Because i have obligatory need set that, without it loop for all game and repeat help with elad

Reply:
elad what

Reply:
Other dev but i have test 50 thinks juste  before understund how can works with information have said me elad ^^

Reply:
I will let the other review